sdks/python/apache_beam/runners/direct/transform_evaluator.py

```diff
@@ -826,3 +830,74 @@ def finish_bundle(self):
           None, '', TimeDomain.WATERMARK, WatermarkManager.WATERMARK_POS_INF)
 
     return TransformResult(self, [], [], None, {None: hold})
+
+
+class _ProcessElemenetsEvaluator(_TransformEvaluator):
+  """An evaluator for sdf_direct_runner.ProcessElements transform."""
+
+  DEFAULT_MAX_NUM_OUTPUTS = 100
+  DEFAULT_MAX_DURATION = 1
```

Comment:
<!--new_thread; commit:7549b5c2ebe2ae47af9066eaf97364a27e828ab5; resolved:0-->
Can you please document these flags?